exercises/support.js

```diff
@@ -559,6 +559,7 @@ class Task {
   }
 }
 
+/* eslint-disable max-len, global-require, no-return-assign, no-param-reassign */
```

Comment:
This file came as is but threw some linting errors. As I will not touch it I just ignore these errors from now on.